README.md

```diff
@@ -292,8 +292,9 @@ The obvious workaround is `Red(fmt.Sprintf("%T", some))`
 
 ### Windows
 
-The Aurora provides ANSI colors only. So, there are not supports and workarounds for OS Windows.
-Check out this commetns to find a way
+The Aurora provides ANSI colors only, so there is no support or workarounds for Windows.
+Check out these comments to learn why:
```

Comment:
Hi, man. Thanks for the PR. I have some questions.

> to learn why

The comments are about workarounds. Hm, other side, we can't use the "workaround" word, because it's stylistically illiterate (because of "no workarounds" above).

The meaning is "take a look to find a workaround". And it's for uses who want Windows support (not learning anything).

Actually, I don't know English language and culture enough to resolve.

Reply:
Ah I understand the intent. I will update 👍

Reply:
@logrusorgru updated